src/components/ShopItem/ShopItem.js

```diff
@@ -1,51 +1,51 @@
 
 class ShopItem extends Component {
-    render() {
-        const item = this.props.item;
-        return (
-            <div className="item">
-                <div>
-                    <img src={item.img} alt=""/>
-                </div>
-                <div>
-                    <h4 className="item-header">{item.name}</h4>
-                    <p><small>by: {item.seller}</small></p>
-                    <div className="item-description">
-                    <div>
-                            <p>${item.price}</p>
-                            <p><small>only {item.stock} left in stock - order soon</small></p>
-                            <button onClick={() => this.props.addToCart(item.key)}>
-                                <i className="fa fa-shopping-cart"></i>
-                                <span>add to cart</span>
-                            </button>
-                    </div>
-                    <div>
-                            <Rating
-                                className="ratings"    
-                                empty="fa fa-star-o"
-                                full="fa fa-star"
-                                placeholder="fa fa-star"
-                                placeholderRate={item.star}
-                                readonly
-                            ></Rating>
-                            <h4>Features</h4>
-                            <ul>
-                                {
-                                    item.features.map(ftr => <li key={ftr.description}>{ftr.description}: <strong>{ftr.value}</strong> </li> )    
-                            }
-                            </ul>    
-                    </div>
-                </div>
-                </div>
-                
+  addToCartHandler() {
+    this.props.updateOrderSummary(this.props.item);
+  }
+  render () {
+    const item = this.props.item;
+    return (
+      <li className="item">
+        <div className="image-container">
+          <img src={item.img} alt={item.name}/>
+        </div>
+        <div className="item-details">
+          <h1>{item.name}</h1>
+          <Rating
+            className="ratings"    
+            empty="fa fa-star-o"
+            full="fa fa-star"
+            placeholder="fa fa-star"
+            placeholderRate={item.star}
+            readonly
+          ></Rating>
+          <p>by: {item.seller}</p>
+          <div className="l-display-flex">
+            <div className="half-width">
+              <p className="l-margin-top">${item.price}</p>
+              <p>only {item.stock} left in stock - order soon</p>
             </div>
-        );
-    }
+            { item.features.length > 0 &&
+              <div className="half-width">
+                <h2>Features</h2>
+                <ul>
+                  {
+                    item.features.map(feature => <li key={feature.description}>{feature.description}:{feature.value}</li>)
```

Comment:
Function too long  , please make it readable.
user comments to understand the code implementation intent.